specification/paths/CalculateLandedCosts.json

```diff
@@ -87,7 +89,19 @@
               ],
               "properties": {
                 "data": {
-                  "$ref": "#/components/schemas/Shipment"
+                  "type": "object",
+                  "required": [
+                    "total_tax",
+                    "total_duty"
+                  ],
+                  "properties": {
```

Comment:
We also output the items with individual `tax` and `duty` and actually also the `hs_code` if we needed to look it up.

Reply:
I think we should then rediscuss this on monday (i'm a bit gaar atm). Because that doesnt add up with the output we discussed earlier: ```data: {
  total_tax: {
    amount:
    currency:
  },
  total_duty: {
    amount:
    currency:
  }
}```